tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -32,36 +32,89 @@ static struct fs_mount_t test_fs_mnt_unsupported_fs = {
 		.fs_data = &test_data,
 };
 
-static struct fs_mount_t test_fs_mnt_invalid_root = {
+/* invalid name of mount point, not start with '/' */
+static struct fs_mount_t test_fs_mnt_invalid_root_1 = {
 		.type = TEST_FS_2,
 		.mnt_point = "SDA:",
 		.fs_data = &test_data,
 };
 
+/* length of the name of mount point is too short */
+static struct fs_mount_t test_fs_mnt_invalid_root_2 = {
+		.type = TEST_FS_2,
+		.mnt_point = "/",
+		.fs_data = &test_data,
+};
+
+/* NULL mount point */
+static struct fs_mount_t test_fs_mnt_invalid_root_3 = {
+		.type = TEST_FS_2,
+		.mnt_point = NULL,
+		.fs_data = &test_data,
+};
+
 static struct fs_mount_t test_fs_mnt_already_mounted = {
 		.type = TEST_FS_2,
 		.mnt_point = TEST_FS_MNTP,
 		.fs_data = &test_data,
 };
 
+/* for test_fs, name of mount point must end with ':' */
 static struct fs_mount_t test_fs_mnt_invalid_parm = {
 		.type = TEST_FS_2,
 		.mnt_point = "/SDA",
 		.fs_data = &test_data,
 };
 
+#define NOOP_MNTP "/SDCD:"
 static struct fs_mount_t test_fs_mnt_no_op = {
 		.type = TEST_FS_2,
-		.mnt_point = "/SDA:",
+		.mnt_point = NOOP_MNTP,
 		.fs_data = &test_data,
 };
 
 static struct fs_file_t filep;
+static struct fs_file_t err_filep;
 static const char test_str[] = "hello world!";
 
 /**
  * @brief Test mount interface of filesystem
  *
+ * @details
+ * Test Objective
+ * - Test fs_mount() interface in file system
+ * - Following test cases depend on file systems mounted in this test case
+ *
+ * Testing techniques:
+ * - Interface testing
+ * - function and block box testing
+ * - dynamic analysis and testing
+ *
+ * Prerequisite Conditions:
+ * - N/A
+ *
+ * Input Specifications:
+ * - N/A
+ *
+ * Test Procedure:
+ * -# Mount a file system with NULL parameter
+ * -# Mount a file system unsupported
+ * -# Register a file system and mount it to invalide directories
```

Comment:
"invalide" -> "invalid"; "attempt mounting it at invalid directory paths" maybe...